templates/console/source/lambda/config-version/cf_config_version_exporter/cf_config_version_exporter.py

```diff
@@ -12,33 +12,36 @@
 DDB_LATESTVERSION_TABLE_NAME = os.environ['DDB_LATESTVERSION_TABLE_NAME']
 DDB_SNAPSHOT_TABLE_NAME = os.environ['DDB_SNAPSHOT_TABLE_NAME']
 
-# TODO: for local debug purpose, will remove before release
-# S3_BUCKET = "cloudfrontconfigversions-cloudfrontconfigversions-60jwdz7zg1zi"
-# DDB_VERSION_TABLE_NAME = 'CloudFrontConfigVersionStack-CloudFrontConfigVersionTable6E23F7F5-D8I07GGNBYFJ'
-# DDB_LATESTVERSION_TABLE_NAME = 'CloudFrontConfigVersionStack-CloudFrontConfigLatestVersionTable44770AF8-7LF5V48RKDK0'
 
 log = logging.getLogger()
 log.setLevel('INFO')
 
 
 def lambda_handler(event, context):
     # extract the distribution id from the input
+    distribution_id = get_distributionId(event)
+
+    cf_client = boto3.client('cloudfront')
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

A Boto3 client is initialized from a Lambda handler. To speed up your Boto3 client initialization and minimize the operational cost of your Lambda function, we recommend that you create the client at the level of the module that contains the handler, and then reuse it between invocations.

[Learn more](https://docs.aws.amazon.com/lambda/latest/dg/best-practices.html#function-code)

Similar issue at line number 25.